### ISSUE TITLE
Use isbataviainstance where check classes are needed.

### ISSUE THREAD

**Issue by ramiroluz** [pybee/batavia#333]
Rename the function to isbataviainstance.

https://github.com/pybee/batavia/blob/master/batavia/utils.js#L48

Example of use: https://github.com/ramiroluz/batavia/blob/8708f51d7b183d632c5465a26481675d5d52c7a4/batavia/types/Bool.js#L301

I can do that, just let me know if you don't think it is a good idea.

**Comment by freakboy3742** [comment]
Yes -  this looks like a really good idea. If #335 is any indication, this will clean up the code significantly.
